app/src/main/res/layout/vocable_confirmation_dialog.xml

```diff
@@ -0,0 +1,74 @@
+
+    <TextView
+        android:id="@+id/dialog_title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dialog_text_margin"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Leaving the app" />
+
+    <TextView
+        android:id="@+id/dialog_message"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/dialog_title"
+        app:layout_constraintTop_toBottomOf="@+id/dialog_title"
+        tools:text="You're about to be taken outside of the app. You may lose head tracking control." />
+
+    <com.willowtree.vocable.customviews.ActionButton
+        android:id="@+id/dialog_positive_button"
+        android:layout_width="@dimen/dialog_button_width"
+        android:layout_height="@dimen/dialog_button_height"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_button_end_margin"
+        android:layout_marginBottom="@dimen/dialog_button_end_margin"
+        android:background="@drawable/vocable_dialog_button_background"
+        android:gravity="center"
+        android:textAllCaps="true"
+        android:textColor="@color/colorPrimary"
+        android:textSize="@dimen/dialog_button_text_size"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/dialog_message"
+        tools:text="Continue" />
+
+    <com.willowtree.vocable.customviews.ActionButton
+        android:id="@+id/dialog_negative_button"
+        android:layout_width="@dimen/dialog_button_width"
+        android:layout_height="@dimen/dialog_button_height"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_button_end_margin"
+        android:layout_marginBottom="@dimen/dialog_button_end_margin"
+        android:background="@drawable/vocable_dialog_button_background"
+        android:gravity="center"
+        android:textAllCaps="true"
+        android:textColor="@color/colorPrimary"
+        android:textSize="@dimen/dialog_button_text_size"
+        android:textStyle="bold"
```

Comment:
Same deal here with the action buttons and common attributes - worth extracting to a style?